Framework/Test_Framework_UnitTest.rb

```diff
@@ -173,48 +186,48 @@ def test_send_invalid_keys
 
   def test_scrolling
     create_testframe_instance = SeleniumFrameWork.new()
-    url = "file://src/Test_Framework.html"
+    url = "file:///src/Test_Framework.html"
     create_testframe_instance.open_site(url)
     assert_equal("Scrolled", create_testframe_instance.scroll(1))
     create_testframe_instance.close_site()
   end
 
   def test_invalid_scrolling
     create_testframe_instance = SeleniumFrameWork.new()
-    url = "file://src/Test_Framework.html"
+    url = "file:///src/Test_Framework.html"
     create_testframe_instance.open_site(url)
     assert_equal("Not Scrolled", create_testframe_instance.scroll("a"))
     create_testframe_instance.close_site()
   end
 
   def test_select_option
     create_testframe_instance = SeleniumFrameWork.new()
-    url = "file://src/Test_Framework.html"
+    url = "file:///src/Test_Framework.html"
     create_testframe_instance.open_site(url)
     assert_equal("female", create_testframe_instance.select_option("id", "gender", "Female"))
     create_testframe_instance.close_site()
   end
 
   def test_select_invalid_option
     create_testframe_instance = SeleniumFrameWork.new()
-    url = "file://src/Test_Framework.html"
+    url = "file:///src/Test_Framework.html"
     create_testframe_instance.open_site(url)
```

Comment:
Each unit test should test only one function. This unit test uses the open_site method of the framework. You can use the driver methods to perform other actions than the ones we're testing.

Reply:
Change other unit tests accordingly